sqlmesh/core/loader.py

```diff
@@ -404,32 +411,64 @@ def _load() -> Model:
                     except SqlglotError as ex:
                         raise ConfigError(f"Failed to parse a model definition at '{path}': {ex}.")
 
-                return load_sql_based_model(
-                    expressions,
-                    defaults=self.config.model_defaults.dict(),
-                    macros=macros,
-                    jinja_macros=jinja_macros,
-                    audit_definitions=audits,
-                    default_audits=self.config.model_defaults.audits,
-                    path=Path(path).absolute(),
-                    module_path=self.config_path,
-                    dialect=self.config.model_defaults.dialect,
-                    time_column_format=self.config.time_column_format,
-                    physical_schema_mapping=self.config.physical_schema_mapping,
-                    project=self.config.project,
-                    default_catalog=self.context.default_catalog,
-                    variables=self._variables,
-                    infer_names=self.config.model_naming.infer_names,
-                    signal_definitions=signals,
-                )
+                model_meta = seq_get(expressions, 0)
+                blueprints = isinstance(model_meta, ModelMeta) and model_meta.meta.get("blueprints")
+
+                loaded_models = []
+                for blueprint in blueprints or [None]:
+                    if not blueprint:
+                        blueprint_variables = {}
+                    elif isinstance(blueprint, (exp.Tuple, exp.Array)):
+                        blueprint_variables = {
+                            # Assumes the syntax: (k := v [, ...]) or [k := v [, ...]]
```

Comment:
What is this syntax?

Reply:
We need to discuss this. I don't like how inconsistent the 2 alternative ways of doing this are. I think we should stick with either one or the other. The key value one looks not bad

Reply:
there is no need for the blueprint syntax, we only need the kv syntax, it's strictly better

Reply:
I guess the advantage of using `blueprints (a, b, c, d)` is that it's more concise. The alternative would be: `blueprints ((blueprint := a), (blueprint := b), (blueprint := c), (blueprint := d))`, which is kinda repetitive.

Reply:
Got rid of the `blueprints (a, b, c, d)` syntax. Only the key-value one is supported now.